src/main/docs/guide/quickStart.adoc

```diff
@@ -0,0 +1,76 @@
+After installing the https://docs.micronaut.io/1.3.3/guide/index.html#buildCLI[cli],
```

Comment:
wow, Micronaut 1.3.3 ?

I think it is better to remove the link. or use `latest`

Reply:
Wow...good catch... changed it to not have a link